.github/workflows/test-package.yml

```diff
@@ -103,6 +103,7 @@ jobs:
     - name: Install datacompy
       run: |
         python -m pip install --upgrade pip
+        python -m pip install "pandas<2.2"
```

Comment:
Is Pandas 2.2 not supported?

Reply:
2.2 is not supported by Ray. Fugue is fine with that version.